NEWS.md

```diff
@@ -36,6 +36,8 @@ plumber 1.0.0
 
 * Added support for promises in endpoints, filters, and hooks. (#248)
 
+* Add support for `#' @plumber` tag that modify plumber router programmatically using expression of the form `function(pr) {....}`. (@meztez and @blairj09, #568)
```

Comment:
Would also like to see this point to an example, perhaps on the website.

```suggestion
* Add support for `#' @plumber` tag to gain programmatic access to the `plumber` router via `function(pr) {....}`. (@meztez and @blairj09, #568)
```

Reply:
```r
#* @plumber
function(pr){
  pr$handle("GET", "/avatartare", function(raw) {raw}, serializer = serializer_json)
}
```